crates/iceberg-catalog/src/api/management/mod.rs

```diff
@@ -1099,6 +1118,10 @@ pub mod v1 {
                     "/warehouse/{warehouse_id}/deleted_tabulars/undrop",
```

Comment:
We don't have a good way to mark this deprecated right?

Reply:
not really, we also cannot easily drop the route with `deleted_tabulars` from the openapi specs as that would be a breaking change. We could duplicate the handler function, mark the deprecated route as deprecated in the api doc & remove it in a later release.

Reply:
UI Issue: https://github.com/lakekeeper/console/issues/101